### PR TITLE
BurstCube Mission page

### PR DESCRIPTION
**Pull request by jracusin**
GCN - Missions - BurstCube

### CODE REVIEW

**Comment by jracusin** [comment]
Addresses issue #2045

**Review by aam97 — APPROVED** [review]

File: app/routes/missions.burstcube/logo.png
Comment (lpsinger) [inline app/routes/missions.burstcube/logo.png:1]: Would you please provide any SVG version of this image?

File: app/routes/missions.burstcube/route.mdx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+**Data Archive:**
+[HEASARC BurstCube Archive](https://heasarc.gsfc.nasa.gov/docs/burstcube/archive/)
+
+[BurstCube](https://science.nasa.gov/mission/burstcube/) is a 6U CubeSat operated by NASA. It monitors the sky, autonomously detecting gamma-ray transients and relaying alerts to the ground via NASA's TDRS space communication network.
+
+<div className="overflow-table">
+
+| Instruments | Energy Range | Field of View | Localization                              |
+| ----------- | ------------ | ------------- | ----------------------------------------- |
+| BurstCube   | 50-1000 keV  | 6 ster        | ~30° radius (statistical, 90% confidence) |
```
Comment (lpsinger) [inline app/routes/missions.burstcube/route.mdx:34]:
```suggestion
| BurstCube   | 50-1000 keV  | 6 sr          | ~30° radius (statistical, 90% confidence) |
```

File: app/routes/missions.burstcube/route.mdx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+example [JSON message](/schema/stable/gcn/notices/burstcube/alert.example.json) files are available to use. See the [Schema Browser](/docs/schema/stable/gcn/notices/burstcube/alert.schema.json) for more information on the properties defined in the schema.
+
+[Detailed Description and Examples](https://github.com/nasa-gcn/gcn-schema/tree/main/gcn/notices/burstcube)
+
+<div className="overflow-table">
+
+| Type             | Contents                 | Latency    |
+| ---------------- | ------------------------ | ---------- |
+| `Alert`          | Detection of a burst     | 5 min      |
+| `Alert` (update) | Localization HEALPix map | 15 minutes |
```
Comment (lpsinger) [inline app/routes/missions.burstcube/route.mdx:52]:
```suggestion
| `Alert` (update) | Localization HEALPix map | 15 min     |
```

File: app/routes/missions.tsx
Comment (lpsinger) [inline app/routes/missions.tsx:1]: It looks like you rearranged the order of the missions. Would you please do that in a separate PR?

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/missions.burstcube/logo.png
Comment (jracusin) [reply app/routes/missions.burstcube/logo.png:1]: We do not have a SVG version of the image, only a PDF, and I've tried to convert to SVG, but have only come up with a very distorted version. If you have a good way to convert, I can send you the pdf.

File: app/routes/missions.tsx
Comment (jracusin) [reply app/routes/missions.tsx:1]: See #2166

**Review by dakota002 — APPROVED** [review]

**Comment by lpsinger** [comment]
Note that this added some broken links because we don't yet have a release of the schema that includes BurstCube.

**Pull request merged**